components/post_view/post_body/post_body.jsx

```diff
@@ -194,7 +200,10 @@ export default class PostBody extends React.PureComponent {
                             className='theme'
                             onClick={this.props.handleCommentClick}
                         >
-                            {message}
+                            <Markdown
+                                message={formatWithRenderer(message, new RemoveMarkdown())}
```

Comment:
This is actually running the message through the Markdown parser/renderer twice since it's doing it once here and once inside the Markdown component. Maybe we could instead pass the renderer as a prop to the Markdown component and then have it pass it into `TextFormatting.formatText`?

Reply:
Sure, I'll address that after QA review.  thanks